app/views/candidate_mailer/tailored_rejection_advice/_teaching_knowledge_tailored_advice.text.erb

```diff
@@ -0,0 +1,13 @@
+<% if @application_choice.tailored_advice_reasons["teaching_knowledge"].present? %>
+  <% if @application_choice.render_tailored_advice_section_headings? %>
+    <% if @application_choice.tailored_advice_reasons['teaching_knowledge'].include? 'subject_knowledge' %>
+## Improve your subject knowledge
+    <% else %>
+## Get classroom experience
+    <% end %>
+  <% end %>
+
+  <% @application_choice.tailored_advice_reasons['teaching_knowledge'].each do |advice_reason_id| %>
+<%= render "candidate_mailer/tailored_rejection_advice/#{advice_reason_id}_tailored_advice" %>
```

Comment:
indentation

Reply:
Markdown is funny. If I indent this, the headings don't render, they show up literally like "## Heading", same with bullets.

Reply:
😱 That's annoying, makes it harder to scan.

OK 👍🏼

Reply:
This one isn't markdown, so this one should be fine right?